azext_edge/edge/params.py

```diff
@@ -1134,6 +1136,59 @@ def load_iotops_arguments(self, _):
             arg_group="Connector",
         )
 
+    with self.argument_context("iot ops schema") as context:
+        context.argument(
+            "schema_name",
+            options_list=["--name", "-n"],
+            help="Schema name.",
+        )
+        context.argument(
+            "schema_registry_name",
+            options_list=["--registry"],
+            help="Schema registry name.",
+        )
+        context.argument(
+            "schema_format",
+            options_list=["--format"],
+            help="Schema format.",
+            arg_type=get_enum_type(SchemaFormat)
+        )
+        context.argument(
+            "schema_type",
+            options_list=["--type"],
+            help="Schema type.",
+            arg_type=get_enum_type(SchemaType)
+        )
+        context.argument(
+            "description",
+            options_list=["--desc"],
+            help="Description for the schema.",
+        )
+        context.argument(
+            "display_name",
+            options_list=["--display-name"],
+            help="Display name for the schema.",
+        )
+        context.argument(
+            "schema_version",
+            options_list=["--version", "--ver"],
+            help="Schema version name.",
+            type=int,
+            arg_group="Version"
+        )
+        context.argument(
+            "schema_content",
+            options_list=["--version-content", "--vc"],
```

Comment:
is there another `schema_content` that isn't version related? `--vc` seems kinda weird for `schema_content` but makes total sense for `schema_version_content` etc

Reply:
nope - will update variables